### PR TITLE
Add Security configuration paragraph

### PR DESCRIPTION
**Pull request by jozefcipa**
Mention option to set jwt secret code via `JWT_SECRET` environment variable

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [x] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: docs/3.0.0-beta.x/plugins/users-permissions.md [hunk 1]
```diff
@@ -435,3 +435,6 @@ You can update these template in the **Email Templates** tab in the admin panel.
   - ...and every other fields that you added manually in the model.
 - `CODE` corresponds to the CODE generated to be able confirm the user email.
 - `URL` is the Strapi backend URL that confirm the code (by default `/auth/email-confirmation`).
+
+## Security configuration
+JWT tokens can be verified and trusted because the information is digitally signed. To sign a token a *secret* is required. By default Strapi generates one that is stored in `./your-app/extensions/users-permissions/config/jwt.json`. This is useful during development but for security reasons it's **recommended** to set a custom token via environment variable `JWT_SECRET` when deploying to production.
```
Comment (derrickmehaffy) [inline docs/3.0.0-beta.x/plugins/users-permissions.md]: `it's **recommended**` Go ahead and break that up and use `it is **recommended**`

Also `via an environment variable`

Comment (jozefcipa) [reply docs/3.0.0-beta.x/plugins/users-permissions.md]: Fixed, thanks 👍

File: docs/3.0.0-beta.x/plugins/users-permissions.md [hunk 2]
```diff
@@ -435,3 +435,10 @@ You can update these template in the **Email Templates** tab in the admin panel.
   - ...and every other fields that you added manually in the model.
 - `CODE` corresponds to the CODE generated to be able confirm the user email.
 - `URL` is the Strapi backend URL that confirm the code (by default `/auth/email-confirmation`).
+
+## Security configuration
+JWT tokens can be verified and trusted because the information is digitally signed. To sign a token a *secret* is required. By default Strapi generates one that is stored in `./your-app/extensions/users-permissions/config/jwt.json`. This is useful during development but for security reasons it is **recommended** to set a custom token via an environment variable `JWT_SECRET` when deploying to production. It is also possible to modify `jwt.json` file to accept `JWT_TOKEN` automatically by doing following ([docs](https://strapi.io/documentation/3.0.0-beta.x/concepts/configurations.html#dynamic-configurations)).
+```
+  "jwtSecret": "${JWT_SECRET}"
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/plugins/users-permissions.md]: should be ` "jwtSecret": "${process.env.JWT_SECRET}" `

Comment (derrickmehaffy) [reply docs/3.0.0-beta.x/plugins/users-permissions.md]: :man_facepalming: can't believe I missed that

Comment (jozefcipa) [reply docs/3.0.0-beta.x/plugins/users-permissions.md]: Oh, good catch 🤦‍♂ thanks

**Pull request merged**